modules/ank/arrow-ank/src/main/kotlin/arrow/ank/interpreter.kt

```diff
@@ -130,8 +130,8 @@ val interpreter: AnkOps = object : AnkOps {
     val result: Tuple3<SnippetParserState, Sequence<String>, Sequence<Snippet>> =
       content
         .fold(
-          Tuple3(SnippetParserState.Searching, emptySequence(), emptySequence())
-        ) { (state, lines, snippets), line ->
+          Tuple3(SnippetParserState.Searching as SnippetParserState, emptySequence(), emptySequence())
```

Comment:
Is Inference not working as expected when the inline modifier is removed? If so that looks like a compiler bug

Reply:
That is true